examples/pipelines/simple_pipeline/components/image_cropping/fondant_component.yaml

```diff
@@ -0,0 +1,30 @@
+name: Image cropping
+description: Component that removes single-colored borders around images and crops them appropriately
+image: ghcr.io/ml6team/image_cropping:latest
+
+input_subsets:
+  images:
+    fields:
+      data:
+        type: binary
+
+output_subsets:
+  images_crop:
+    fields:
+      data:
+        type: binary
+      width:
+        type: int16
+      height:
+        type: int16
+
+args:
+  scale:
+    description: Scale parameter used for detecting borders
```

Comment:
These arguments are not completely clear to me. Can we explain them in a bit more detail? Either here or in the readme.

Reply:
I removed the first argument since it's not that important and added some information about the offset